js/deribit.js

```diff
@@ -95,30 +95,35 @@ module.exports = class deribit extends Exchange {
                         // Subscription management
                         'subscribe',
                         'unsubscribe',
+                        'unsubscribe_all',
                         // Account management
                         'get_announcements',
                         // Market data
                         'get_book_summary_by_currency',
                         'get_book_summary_by_instrument',
                         'get_contract_size',
                         'get_currencies',
+                        'get_delivery_prices',
                         'get_funding_chart_data',
                         'get_funding_rate_history',
                         'get_funding_rate_value',
                         'get_historical_volatility',
                         'get_index',
                         'get_index_price',
                         'get_index_price_names',
+                        'get_instrument',
                         'get_instruments',
                         'get_last_settlements_by_currency',
                         'get_last_settlements_by_instrument',
                         'get_last_trades_by_currency',
                         'get_last_trades_by_currency_and_time',
                         'get_last_trades_by_instrument',
                         'get_last_trades_by_instrument_and_time',
+                        'get_mark_price_history',
```

Comment:
I was going to add `fetchMarkOHLCV` but they dont return and `OHLCV` for this, it looks like they just return the price with a timestamp

```
{"jsonrpc":"2.0",
  "id": 25,
  "result": [
    [1608142381229,0.5165791606037885],
    [1608142380231,0.5165737855432504],
    [1608142379227,0.5165768236356326]
  ]
}
```